yellowbrick/classifier/prcurve.py

```diff
@@ -350,16 +373,18 @@ def _draw_multiclass(self):
 
         self._draw_ap_score(self.score_[MICRO])
 
-    def _draw_pr_curve(self, recall, precision, label=None):
+    def _draw_pr_curve(self, recall, precision, label=None, color=None):
         """
         Helper function to draw a precision-recall curve with specified settings
         """
         self.ax.step(
-            recall, precision, alpha=self.line_opacity, where="post", label=label
+            recall, precision, alpha=self.line_opacity, where="post",
+            label=label, color=color
         )
         if self.fill_area:
             self.ax.fill_between(
-                recall, precision, step="post", alpha=self.fill_opacity
+                recall, precision, step="post", alpha=self.fill_opacity,
+                color=color
```

Comment:
Should we turn off `fill_area` in the case of multiclass?

Reply:
Right, I think that we should turn off `fill_area` in case of multiclass. [This](https://scikit-learn.org/stable/auto_examples/model_selection/plot_precision_recall.html) sklearn example does not fill the area under PR curve.